### PR TITLE
chore(docs): updated the name of Twitter to X

### PR DESCRIPTION
**Pull request by tvermaashutosh**
## Checklist

- [x] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [x] Description explains motivation and solution
- [x] Tests added (always)
- [x] Docs updated (only required for features)
- [x] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Comment by tvermaashutosh** [comment]
@eladb @staycoolcall911 
I have enhanced/improved the CODE_OF_CONDUCT Guidelines via updating the CODE_OF_CONDUCT.md file. Making it more clear/comprehensible for the users, ensuring that users can easily access and understand the CODE_OF_CONDUCT instructions information.
I have also corrected the minor grammatical errors etc. that were present in the file previously.

Kindly review my PR and merge it please.

File: examples/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ _Add yours here!_
 ### Tweet queue
 
 * **Source code:** [./tweet-queue.w](./tweet-queue.w)
-* **Description:** This is a Wing app that lets you get a feed of Twitter search results into a queue. It works by periodically polling the freely available Twitter Standard Search API and sending all new tweets to the queue.
+* **Description:** This is a Wing app that lets you get a feed of X search results into a queue. It works by periodically polling the freely available X Standard Search API and sending all new tweets to the queue.
```
Comment (MarkMcCulloh) [inline examples/README.md:60]: Arguably this whole comment can be deleted. afaik the search API is no longer free anyways, and this was never implemented here.

Comment (tvermaashutosh) [reply examples/README.md:60]: OK @MarkMcCulloh 
I'll quickly delete it.

**Comment by MarkMcCulloh** [comment]
Thanks for your contribution, please review the the PR checklist to format your PR title correctly.

File: docs/contributing/999-rfcs/2022-05-28-winglang-reqs.md [hunk 2]
```diff
@@ -1013,6 +1013,6 @@ This is a list of features we will consider for wing as it evolves:-
 * [Hindley–Milner type system](https://en.wikipedia.org/wiki/Hindley%E2%80%93Milner_type_system)
 * [Funarg problem](https://en.wikipedia.org/wiki/Funarg_problem)
 * [Multi-stage programming](https://en.wikipedia.org/wiki/Multi-stage_programming)
-* [Compiler Books](https://twitter.com/mikhailshilkov/status/1492801675068952576?s=21)
+* [Compiler Books](https://x.com/mikhailshilkov/status/1492801675068952576?s=21)
```
[the remark this answers is not in the document]
Comment (tvermaashutosh) [reply docs/contributing/999-rfcs/2022-05-28-winglang-reqs.md]: I have made this change as per your request.

File: examples/README.md (hunk 1 above)
Comment (tvermaashutosh) [reply examples/README.md]: I have made this change as per your request.

**Comment by tvermaashutosh** [comment]
Is it up to the mark to get merged?

File: docs/contributing/999-rfcs/2022-05-28-winglang-reqs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (staycoolcall911) [reply docs/contributing/999-rfcs/2022-05-28-winglang-reqs.md]: @MarkMcCulloh - the link to nitter.net leads to a Privacy Error page for me. I'd suggest to leave it x.com as @tvermaashutosh suggested:
image

[the remark this answers is not in the document]
Comment (tvermaashutosh) [reply docs/contributing/999-rfcs/2022-05-28-winglang-reqs.md:1016]: @staycoolcall911 
I have updated the `Compiler Books link` from `https://nitter.net/mikhailshilkov/status/1492801675068952576` to `https://x.com/mikhailshilkov/status/1492801675068952576?s=21`.

I request you to kindly review my PR and merge it please.

**Review by staycoolcall911 — APPROVED** [review]

**Pull request merged**